### ISSUE TITLE
Random Mega Forms

### ISSUE THREAD

**Issue by uscdude92** [kwsch/pk3DS#297]
Hi, I caught a random Mega Venasaur as my wild encounter, as I chose this option, yet the Venasaur resorted to its normal form after saving my game and turning it off? Any idea why this happens?

**Comment by sora10pls** [comment]
The form reverts when it's left in your party after closing the game. To prevent this from happening, place it in the PC before closing the game.

This is normal behavior from the games and is not a bug with pk3DS, this can be closed.

**Comment by bobalife808** [comment]
If I happened to leave them in my party, and they reverted but then I put them back and saved and it didn't revert back, is there any way to fix that?

**Comment by kwsch** [comment]
@bobalife808 no, the game never undoes this change. Mitigations are already listed in the above comment chain.
